apps/deploy_monitoring/src/monitors/deploy.rs

```diff
@@ -0,0 +1,250 @@
+// Copyright (c) SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+#![forbid(unsafe_code)]
+
+use std::fs::File;
+use std::io::Write;
+use std::io::{BufRead, BufReader};
+use std::path::Path;
+use std::path::PathBuf;
+
+use chrono::Utc;
+use failure::bail;
+use shiplift::Docker;
+use slog::{info, warn, Logger};
+use zip::write::ZipWriter;
+
+use crate::deploy_with_compose::{
+    restart_sandbox, restart_stack, shutdown_and_update, shutdown_and_update_sandbox, DEBUGGER_PORT,
+};
+use crate::image::{
+    local_hash, remote_hash, DeployMonitoringContainer, Explorer, Sandbox, TezedgeDebugger,
+};
+
+use crate::monitors::TEZEDGE_VOLUME_PATH;
+use crate::node::TezedgeNode;
+use crate::slack::SlackServer;
+
+const TEZEDGE_NODE_P2P_PORT: u16 = 9732;
```

Comment:
@adonagy 
also port constant, I would put them together in one place, maybe `mod constants`